asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -313,7 +314,31 @@ const ExportPage = (props) => {
                     </Button>
                   </span>
                 </Tooltip>
-              </Box>
+              </Box>.
+
+<Box 
+  display="flex"
+  justifyContent="center"
+  alignItems="center"
+  height="100%"  // This ensures the container takes the full height of its parent
+>            
+  <Box 
+    border={2} 
+    borderColor="warning.main" 
+    borderRadius={3} 
+    padding={2} 
+    bgcolor="warning.lighter" 
+    maxWidth="600px"  // or whatever maximum width you prefer
+  >
+    <TypographySubtitle1Medium>To Cite ASReview</TypographySubtitle1Medium>
+    <Typography variant="body2" sx={{ color: "text.secondary" }}>
+      ASReview LAB developers. (2023). ASReview LAB - 
+      A tool for AI-assisted systematic reviews (v1.3rc1). Zenodo.  
+      <Link href="https://doi.org/10.5281/zenodo.8297019">https://doi.org/10.5281/zenodo.8297019</Link>
+    </Typography>
+  </Box>
+  </Box>
+
```

Comment:
Can you prettify this file (only this file)? https://github.com/asreview/asreview/blob/master/DEVELOPMENT.md#formatting-and-linting-1

Reply:
Done